packages/security/package.json

```diff
@@ -3,6 +3,7 @@
   "private": true,
   "version": "0.0.0",
   "license": "Apache-2.0",
+  "main": "src/index.js",
```

Comment:
should the `main` not refer to the dist entrypoint, and have a `module` equivalent? https://github.com/carbon-design-system/carbon/blob/master/packages/react/package.json#L6-L7